app/services/valid_test_data_generators/applications_populater.rb

```diff
@@ -39,9 +39,7 @@ def initialize(lead_provider:, cohort:, number_of_participants:, logger: Rails.l
 
     def create_participants!
```

Comment:
Why is the applications populater changing as part of this?

Reply:
As we're creating seed data for different cohorts, validations related to dates get triggered for declarations. I had to move the `travel_to` so we create declarations that are after `schedule.applies_from`.

Reply:
Makes sense 👍 I think we need to keep some of the randomness here though; so previously this was creating participants on random dates over the last 3 years whereas now the seeds would all have the same created at?

Reply:
Sure, I've tried to maintain some randomness with this line:
https://github.com/DFE-Digital/npq-registration/pull/1544/files#diff-38f0cbe9d749dd85fe13a256c7c104dfc5a070dbf1a7e0e205b6c9b80df3ff19R61